tests/torch/data/reference_graphs/quantized/synthetic_model/MHA.dot

```diff
@@ -11,71 +11,61 @@ strict digraph  {
 "9 TestModel/MultiheadAttention[_layer]/SymmetricQuantizer/symmetric_quantize_1" [id=9, type=symmetric_quantize];
```

Comment:
@ljaljushkin looks like the MHA in torch has changed. The new quantizer setup looks OK to me, but please check as well.

Reply:
Looks good, as there's no sum operation and quantizers can be merged.
![image](https://github.com/openvinotoolkit/nncf/assets/4014476/cd46bc5f-2725-4214-b9bf-892d4169b38c)
![image](https://github.com/openvinotoolkit/nncf/assets/4014476/faf5e2c0-29a6-40fc-bd2b-dcb054f05ecb)